configure.ac

```diff
@@ -17,7 +17,7 @@ RPM_RELEASE=0
 AC_SUBST(RPM_RELEASE)
 if test -d ".git" ; then
   AC_CHECK_PROG(GIT_CHECK, git, yes)
-  if test x"$GIT_CHECK" = x"yes"; then
+  if test x"$GIT_CHECK" = x"yes" -a x"$freebsd" != x"yes"; then
```

Comment:
@wjwithagen i think we can leave it unchanged. yes, FreeBSD does not use rpm package manager. but it does not hurt to have the `RPM_RELEASE`.